server/src/infra/repositories/user.repository.ts

```diff
@@ -101,11 +105,37 @@ export class UserRepository implements IUserRepository {
       stat.photos = Number(stat.photos);
       stat.videos = Number(stat.videos);
       stat.usage = Number(stat.usage);
+      stat.quotaSizeInBytes = Number(stat.quotaSizeInBytes) || null;
```

Comment:
Do we need to convert to number here? Looks like we do it above too, just not sure why we don't have the normal types here.

Reply:
There is a bunch of manual parsing for all number columns and types since they come back as strings. This might have changed in a recent PR, I'm not sure. If so, we can eliminate all of this.